src/v2/Apps/Consign/Routes/MarketingLanding/Components/__tests__/CtaBanner.jest.tsx

```diff
@@ -0,0 +1,100 @@
+import { fireEvent, render, screen } from "@testing-library/react"
+import { MockBoot } from "v2/DevTools"
+import { useTracking } from "v2/System/Analytics"
+import { Breakpoint } from "v2/Utils/Responsive"
+import { CtaBannerContent } from "../CtaBanner"
+
+jest.mock("v2/System/Analytics/useTracking")
+
+jest.mock("v2/System/Analytics/AnalyticsContext", () => ({
+  ...jest.requireActual("System/Analytics/AnalyticsContext"),
+  useAnalyticsContext: jest.fn(() => ({
+    contextPageOwnerType: "sell",
+  })),
+}))
+
+const trackEvent = useTracking as jest.Mock
+
+const renderWithBreakpoint = (breakpoint: Breakpoint = "lg") =>
+  render(
+    <MockBoot breakpoint={breakpoint}>
+      <CtaBannerContent />
+    </MockBoot>
+  )
+
+const sharedTests = () => {
+  it("tracks get in touch click", () => {
+    fireEvent.click(screen.getByText("Get in Touch"))
+
+    expect(trackEvent).toHaveBeenCalled()
+    expect(trackEvent).toHaveBeenCalledWith({
```

Comment:
suggestion (non-blocking): It could be good to test for the presence of the `user_id` field if the user is authenticated. (Would that field be `undefined` if they are not authenticated?)

Reply:
Added. 

> Would that field be undefined if they are not authenticated?

Yes, and it's a reason why I don't check it in the test in this case, it's empty.